src/cryptography/x509/base.py

```diff
@@ -183,6 +187,51 @@ def public_bytes(self, encoding):
         Serializes the certificate to PEM or DER format.
         """
 
+    def verify(self, intermediates, trusted_roots, cert_verif_cb):
+        """
+        Find chains of trust from `self` to any certificate in `trusted_roots`,
+        using certificates from `intermediates`.
+        :param intermediates: list of Certificate objects to build chains from
+        :param trusted_roots: list of Certificate objects which are trusted
+        :param cert_verif_cb: ignored for now. Will take (chain: List(Certificate), position: int)
```

Comment:
```suggestion
        :param cert_verif_cb: Will take (chain: List(Certificate), position: int)
```
cert_verif_cb does not look to be ignored anymore